app/kumactl/cmd/generate/generate_zoneingress_token.go

```diff
@@ -23,7 +23,10 @@ func NewGenerateZoneIngressTokenCmd(pctx *kumactl_cmd.RootContext) *cobra.Comman
 	cmd := &cobra.Command{
 		Use:   "zone-ingress-token",
 		Short: "Generate Zone Ingress Token",
-		Long:  `Generate Zone Ingress Token that is used to prove Zone Ingress identity.`,
+		Long: `Generate Zone Ingress Token that is used to prove Zone Ingress identity.
+
+DEPRECATED: Use kumactl generate zone-token --scope=ingress instead.
```

Comment:
Why Not use `cobra.Command.Deprecated` ? It also has the advantage of not listing the option anymore to user in help which avoids confusion.

Reply:
Tried that, but it will break stuff
```
❯❯❯ kumactl generate zone-ingress-token --zone=east > /tmp/asdf

❯❯❯ cat /tmp/asdf
Command "zone-ingress-token" is deprecated, Use kumactl generate zone-token --scope=ingress instead.
eyJhbGciOiJSUzI1NiIsImtpZCI6IjEiLCJ0eXAiOiJKV1QifQ.eyJab25lIjoiZWFzdCIsImV4cCI6MTk4MDQxMTM1MCwibmJmIjoxNjY1MDUxMDUwLCJpYXQiOjE2NjUwNTEzNTAsImp0aSI6Ijk1YzA3MTU0LWJjYjYtNGEyMi05NmY3LTY2NGFkM2ZmNmI5YSJ9.hU1FfS9hVtodkLB4nwX7uq6vBP7ycU9zlmsixxiUo3i3k_m-FgoNmek38V07aHuiuOpIP2Ep_PvJ3zGROtLLaiOzDd1Yr8HMo6SlDCiskfhH3csrjVLfPd7XBx2Q1-hpazUIELh4kBVF0FjXM-kW_f6rHpuyKrIdbgY4-yJVRscYtW9FgYnrD0iwYjMfmaFh8ykXUt7uwS4JKrUa4RplC5Jvdcuulk5EZy7Vq4cN4IBrIn2J6aDSBwQuNehjc-VxQ1ea4QQhPa0Rw06lHKoUyUyX5IGye7E1FQ8hs0qtE3xW4Q-IbXrWRXGVYYVemhpnZrLYYIBUlGK8tH3tASxj8w⏎
```